### PR TITLE
chore(log): fix usage of clog

### PR DESCRIPTION
**Pull request by stormqueen1990**

### CODE REVIEW

**Review by imjasonh — DISMISSED** [review]

File: internal/harnesses/container/container.go [hunk 1]
```diff
@@ -47,7 +47,8 @@ func (h *container) Destroy(ctx context.Context) error {
 // StepFn implements types.Harness.
 func (h *container) StepFn(config types.StepConfig) types.StepFn {
 	return func(ctx context.Context) (context.Context, error) {
-		log.Info(ctx, "stepping in container", "command", config.Command)
+		log := clog.FromContext(ctx)
+		log.Infof("stepping in container with command %s", config.Command)
```
Comment (joshrwolf) [inline internal/harnesses/container/container.go:51]: Can you keep this as Info() so it correctly attaches the command? %s is going to have a hard time rendering the arbitrary commands

File: internal/harnesses/container/container.go [hunk 2]
```diff
@@ -60,7 +61,7 @@ func (h *container) StepFn(config types.StepConfig) types.StepFn {
 		if err != nil {
 			return ctx, err
 		}
-		log.Info(ctx, "finished stepping in container", "command", config.Command, "out", string(out))
+		log.Infof("finished stepping in container with command %s, out: %s", config.Command, string(out))
```
Comment (joshrwolf) [inline internal/harnesses/container/container.go:64]: Same here

**Comment by joshrwolf** [comment]
Real quick fixes otherwise lgtm! Thanks for fixing this 🙏

File: internal/harnesses/container/container.go (hunk 1 above)
Comment (stormqueen1990) [reply internal/harnesses/container/container.go:51]: `Info()` won't work because it doesn't plumb the context through

**Review by joshrwolf — APPROVED** [review]

**Pull request merged**